src/client/fuse_ll.cc

```diff
@@ -1122,6 +1122,20 @@ static void fuse_ll_ioctl(fuse_req_t req, fuse_ino_t ino,
       fuse_reply_ioctl(req, 0, arg, sizeof(*arg));
     }
     break;
+    case FS_IOC_GETFLAGS: {
+      generic_dout(0) << __FILE__ << ":" << __LINE__ << ": FS_IOC_GETFLAGS ioctl" << dendl;
+
+      int file_attr = 0;
+      if (out_bufsz < sizeof(file_attr)) {
+          fuse_reply_err(req, ERANGE);
+          break;
+      }
+
+      Fh *fh = (Fh*)fi->fh;
+      cfuse->client->ll_handle_ioctl(fh->inode.get(), FS_IOC_GETFLAGS, &file_attr);
```

Comment:
We're already handling ioctl cmds here no need to do it again later in `ll_handle_ioctl`.

Can you make it call something like `get_inode_flags`?

Reply:
Right, now I see that client just needs to fill the flags. Fixed.
I wonder whether the client needs just to return i_flags or to set explicitly bits that we support right now (e.g. FS_ENCRYPT_FL) ?